contrib/opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -162,14 +162,40 @@ def _find_timecode_mobs(item):
     return mobs
 
 
+def timecode_values_are_same(timecodes):
+    """
+    We can have a valid AAF with multiple timecode objects (for example
+    an auxTC24 value got added via the Avid Bin column). But as long as they
+    have the same start and length values, we can use them, treat them as 'same'
```

Comment:
we've been trying to move comments away from using 'we', to be more direct:

```suggestion
    An AAF can have multiple timecode objects (for example an auxTC24 value
    that got added via the Avid Bin column). As long as they have the same
    start and length values, they can be treated as being the same.
```